src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -498,6 +498,22 @@ abstract class PipelineTargets(config: Config) {
       config
     )
 
+    lazy private[overwatch] val verboseAuditTarget: PipelineTable = PipelineTable(
+      name = "verboseAuditTarget_gold",
+      _keys = Array("organization_id", "timestamp", "unixTimeMS"),
+      config,
+      _mode = WriteMode.merge,
```

Comment:
Why MERGE? Merge is very expensive and the actions are immutable right? By removing dbsql -- you can get rid of potentially getting a start in overwatch run 1 and a finish in run 2 and greatly simplify this. All of the DBSQL query information is captured in SQL_query_history table (and more); thus it doesn't need to be captured here.